File: packages/transform/src/features/http/CreateHttpAssertFormDataTransformer.ts
import { HttpAssertFormDataProgrammer } from "@typia/core";

import { ITransformProps } from "../../ITransformProps";
import { GenericTransformer } from "../../internal/GenericTransformer";

export namespace CreateHttpAssertFormDataTransformer {
  export const transform = (props: ITransformProps) =>
    GenericTransformer.factory({
      ...props,
      method: "http.createAssertFormData",
      write: HttpAssertFormDataProgrammer.write,
    });
}

File: packages/transform/src/features/http/CreateHttpAssertFormDataTransformer.go
package http

import (
	corehttp "github.com/samchon/typia/packages/core/src/programmers/http"
	internal "github.com/samchon/typia/packages/transform/src/internal"
)

var CreateHttpAssertFormDataTransformer = createHttpAssertFormDataTransformerNamespace{}

type createHttpAssertFormDataTransformerNamespace struct{}

func (createHttpAssertFormDataTransformerNamespace) Transform(call internal.ITypiaCallExpression) string {
	return corehttp.HttpAssertFormDataProgrammer.Write(call.TypeText)
}
